template.rb

```diff
@@ -177,6 +177,9 @@ def render_file(filename)
 
           config.web_console.permissions = whitelisted_ips
```

Comment:
Do we need to do something here for `web_console` like we did below on L181 for `better_errors`?

Reply:
@raghubetina `web_console` appears with no additional configuration. I believe if an IP is whitelisted through `BetterErrors::...allow_ip!` then `web_console` will also allow it.